roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -88,14 +88,71 @@
     name: 'install_yamls_makes'
     tasks_from: 'make_edpm_baremetal_compute'
 
-- name: Deploy EDPM with baremetal
+- name: Prepare OpenStack Dataplane NodeSet CR
   vars:
-    make_edpm_deploy_baremetal_env: "{{ cifmw_edpm_deploy_baremetal_common_env |
+    make_edpm_deploy_baremetal_prep_env: "{{ cifmw_edpm_deploy_baremetal_common_env |
       combine(cifmw_edpm_deploy_baremetal_make_openstack_env)}}"
-    make_edpm_deploy_baremetal_dryrun: "{{ cifmw_edpm_deploy_baremetal_dry_run }}"
+    make_edpm_deploy_baremetal_prep_dryrun: "{{ cifmw_edpm_deploy_baremetal_dry_run }}"
   ansible.builtin.include_role:
     name: 'install_yamls_makes'
-    tasks_from: 'make_edpm_deploy_baremetal'
+    tasks_from: 'make_edpm_deploy_baremetal_prep'
+
+- name: Create repo-setup-downstream OpenStackDataPlaneService
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command:
+    cmd: >-
+      oc apply -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+      -f "{{ cifmw_installyamls_repos }}/devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup_downstream.yaml"
```

Comment:
we should change the service name to  `repo-setup` so it will just patch the existing `repo-setup`